### PR TITLE
[Notifier] [Bridge] [KazInfoTeh] added bridge documentation

### PR DESCRIPTION
**Pull request by taranovegor**
Add Notifier bridge for kazinfoteh.kz

symfony/symfony#44360

### CODE REVIEW

File: notifier.rst [hunk 1]
```diff
@@ -87,6 +87,7 @@ Telnyx          ``symfony/telnyx-notifier``           ``telnyx://API_KEY@default
 TurboSms        ``symfony/turbo-sms-notifier``        ``turbosms://AUTH_TOKEN@default?from=FROM``
 Twilio          ``symfony/twilio-notifier``           ``twilio://SID:TOKEN@default?from=FROM``
 Yunpian         ``symfony/yunpian-notifier``          ``yunpian://APIKEY@default``
+KazInfoTeh      ``symfony/kazinfoteh-notifier``       ``kazinfoteh://username:password@default?sender=FROM``
```
Comment (OskarStark) [inline notifier.rst]: Please move above LightSms to keep alphanumeric order, thanks

and

```suggestion
KazInfoTeh      ``symfony/kazinfoteh-notifier``       ``kazinfoteh://USERNAME:PASSWORD@default?sender=FROM``
```

**Review by OskarStark — CHANGES_REQUESTED** [review]
Please add a `versionadded:: 6.1` block, thanks

**Comment by OskarStark** [comment]
Please target `6.1` branch

File: notifier.rst (hunk 1 above)
Comment (OskarStark) [reply notifier.rst]: @fabpot I guess the correct package name would be:
```diff
- symfony/kazinfothe-notifier
+ symfony/kaz-info-the-notifier
```

Can you please confirm?

Comment (taranovegor) [reply notifier.rst]: The correct spelling would be `symfony/kaz-info-teh-notifier`. `teh` it's abbreviation from `technology` in Russian.

Comment (OskarStark) [reply notifier.rst]: Sorry ofc you are right I misspelled it 👍🏻

**Comment by OskarStark** [comment]
Please add a versionadded:: 6.1 block, thanks

You can check 5.4 branch for reference

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
This branch needs a rebase please

File: notifier.rst [hunk 2]
```diff
@@ -90,7 +91,7 @@ Yunpian         ``symfony/yunpian-notifier``          ``yunpian://APIKEY@default
 
 .. versionadded:: 6.1
 
-    The 46elks and OrangeSms integrations were introduced in Symfony 6.1.
+    The KazInfoTeh, 46elks and OrangeSms integrations were introduced in Symfony 6.1.
```
Comment (OskarStark) [inline notifier.rst]:
```suggestion
    The 46elk, OrangeSms and KazInfoTeh integrations were introduced in Symfony 6.1.
```

Please move new bridges to the end, thanks

Comment (taranovegor) [reply notifier.rst]: was replaced

**Review by OskarStark — APPROVED** [review]

File: notifier.rst (hunk 2 above)
Comment (OskarStark) [reply notifier.rst]: thanks

**Comment by OskarStark** [comment]
Thanks for your work on this new feature!

**Pull request merged**